### DISCUSSION TITLE
Possible to alert on values?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#47] [Q&A] [answered]
Hello,

I doubt this is even possible, but can we alert on cells at all?

Thanks

**Comment by marcusolsson** [comment]
It should already be possible. Let me know if doesn't work for you and I'll look into why.

**Reply by G0nz0uk** [reply]
I don’t seem to get an alert tab Marcus, do you?

**Comment by G0nz0uk** [comment, marked as answer]
Hi Marcus, do you have a screenshot on this please then I can check what my I can’t see alerts?

**Reply by marcusolsson** [reply]
Seems to be working for me. You can only create alerts using the Graph panel, and your CSV data must have a Time and a Number field, i.e. it should be a time series.

<img width="1595" alt="Screenshot 2021-03-06 at 13 41 19@2x" src="https://user-images.githubusercontent.com/8396880/110207074-aa6bbb00-7e81-11eb-814c-81b7ffdd88fa.png">
